test/unit/puppetlabs/services/master/master_service_test.clj

```diff
@@ -12,10 +11,12 @@
     [puppetlabs.trapperkeeper.testutils.bootstrap :as tk-testutils]
     [puppetlabs.services.jruby.jruby-testutils :as jruby-testutils]
     [puppetlabs.services.puppet-profiler.puppet-profiler-service :as profiler]
-    [puppetlabs.services.version.version-check-service :as version-check-service]
     [puppetlabs.trapperkeeper.testutils.logging :as logutils]
     [puppetlabs.services.ca.certificate-authority-service :refer [certificate-authority-service]]
-    [me.raynes.fs :as fs]))
+    [puppetlabs.services.ca.certificate-authority-disabled-service :refer [certificate-authority-disabled-service]]
+    [puppetlabs.dujour.version-check :as version-check]
+    [me.raynes.fs :as fs]
+    [puppetlabs.puppetserver.certificate-authority :as ca]))
```

Comment:
The `ca` namespace isn't used anymore now that we've got the disabled CA service, right?